nottingham_covid_modelling/lib/likelihood.py

```diff
@@ -40,8 +42,8 @@ def n_parameters(self):
 class Gauss_LogLikelihood(_LogLikelihood):
     def __init__(self, settings, deaths,
                  parameters_to_optimise=['rho', 'Iinit1', 'lockdown_baseline', 'lockdown_fatigue'], 
-                 travel_data=True):
-        super().__init__(settings, deaths, parameters_to_optimise + ['gaussian_noise_sigma'], travel_data)
+                 travel_data = True, model_func = get_model_solution):
```

Comment:
generally the python style guides say:
vairable assignment is like this:
`var = value`
and parameters being passed is like this (without spaces)
`somefunc(p1=12, p2=x_func, ... )`
just to keep the 2 apart

Reply:
I was careful to leave only the txt files related to the figure, and all the new ones are in one folder. Thanks for the formatting comment!